data/core/units/monsters/Skeletal_Dragon.cfg

```diff
@@ -6,21 +6,21 @@
     race=undead
     image="units/monsters/skeletal-dragon/skeletal-dragon.png"
     image_icon="units/monsters/skeletal-dragon/skeletal-dragon.png~CROP(0,0,160,160)"
-    hitpoints=171
+    hitpoints=98
     movement_type=undeadfly
     movement=5
     experience=200
     level=4
     alignment=chaotic
     advances_to=null
     {AMLA_DEFAULT}
-    cost=288
+    cost=160
     usage=fighter
     [resistance]
         blade=60
         pierce=40
         impact=120
-        fire=120
+        fire=80#since it is the bones of a dragon, retain some fire resistance
```

Comment:
Add a whitespace between the `0` and `#`

Reply:
alright, done